modules/govuk/manifests/apps/content_store.pp

```diff
@@ -89,5 +89,8 @@
     "${title}-ERRBIT_API_KEY":
       varname => 'ERRBIT_API_KEY',
       value   => $errbit_api_key;
+    "${title}-PLEK_SERVICE_TARIFF_URI":
+      varname => 'PLEK_SERVICE_TARIFF_URI',
+      value   => $trade_tariff_uri;
```

Comment:
I don't have much experience using Puppet, but I've had another guess which I hope is more correct... https://github.com/alphagov/govuk-puppet/pull/4952/commits/7a0854e19362ab05cce923fd46174da951bfd703#diff-317c81f58cd20afec981e1e6c339703fR82